enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/rest/ClientErrorResponseDto.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.onboardingserver.provider.rest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import java.util.Objects;
+
+/**
+ * @author Lubos Racansky, lubos.racansky@wultra.com
```

Comment:
This is file and others in this directory aregenerated? I am not sure if we need to specify our names as authors.

Reply:
Well, as I understood, Peter is agains generating for each build, since it is unstable. We agreed with Roman, that we will generate it at lest once.
The main question is whether to keep the code in raw form to have better diff in the future or polish it.
Speaking about packages, I suggest to keep it here as _package private_ not to leak anywhere. It is used by `com.wultra.app.onboardingserver.provider.rest.RestOnboardingProvider`

Reply:
I reconsidered the topic and prefer to polish the generated code (and remove `@Generated`)